<ui version="4.0" >
 <class>StatusWidget</class>
 <widget class="QWidget" name="StatusWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>28</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>StatusWidget</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="leftMargin" >
    <number>4</number>
   </property>
   <property name="topMargin" >
    <number>2</number>
   </property>
   <property name="rightMargin" >
    <number>12</number>
   </property>
   <property name="bottomMargin" >
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="masters_" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Masters:</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="activeMasters_" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>2</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="configs_" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Active Configs:</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="activeConfigs_" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>2</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="runners_" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Runners:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="activeRunners_" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>8</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="streams_" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Streams:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="activeStreams_" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>16</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="pendingLabel_" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Pending:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="pending_" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>0</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="failuresLabel_" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Failures:</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="failures_" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>0</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="now_" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="toolTip" >
      <string>Current system time</string>
     </property>
     <property name="text" >
      <string>23:59:59</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="elapsed_" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="toolTip" >
      <string>Elapsed recording time</string>
     </property>
     <property name="text" >
      <string>T+00:00:00</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
